### DISCUSSION TITLE
Options Picker Sort

### DISCUSSION THREAD

**Discussion by Hystarea** [Budibase/budibase#13556] [Help] [answered]
I am struggling to get an options picker to show in the order that has been requested.

This is a Budibase database using the options field. I have it ordered correctly in the data view, but in the design view, it's sorting it alphabetically.

The options for the field are the number 1,2,3,4,5,6,7,8,9,10,11,12. But when you go to the design view and step the form up it is in the order 1,10,11,12,3,4,5,6,7,8,9. I understand why it is doing this, but I would like it to be ordered differently.

I tried creating a separate table and using a data provider and sorting it using a weighted number, but that did not fix it either.

![data](https://github.com/Budibase/budibase/assets/110421698/0ae5d123-cefd-4de3-bc0e-8c16851ca301)
![design](https://github.com/Budibase/budibase/assets/110421698/cdd16500-a8eb-4a98-b897-683b99b44f82)

**Comment by mikesealey** [comment, marked as answer]
Hi @Hystarea I've had a quick look at this for you.

If you click the little sprocket to open up the menu for that particular input and uncheck "Alphabetical" it should solve your issue.

![image](https://github.com/Budibase/budibase/assets/39269380/faf59b5c-9634-4094-9cf0-81f771df8d65)

With "Alphabetical" unchecked, the dropdown list will display in the order your options are created in the Data tab, so you could do something doesn't follow typical alphabetical or numerical orders
![image](https://github.com/Budibase/budibase/assets/39269380/c11691fc-197a-4af4-b1d5-7e2d78b1f15d)
![image](https://github.com/Budibase/budibase/assets/39269380/37efa09a-3013-4ea9-850a-0a606b9838bc)


Hope this solves your issue!

**Reply by Hystarea** [reply]
Well that was just too easy! I cannot believe I missed that option.

This worked, thanks!

**Reply by mikesealey** [reply]
No problem at all. I've added a little bit to the documentation to cover it. If you see anything else that you feel could be improved, create a new discussion or head to the Budibase discord server. 

[Documentation](https://docs.budibase.com/docs/options)
[Discord](https://github.com/Budibase/budibase/discussions/7905)
